cli/azd/pkg/tools/maven/maven.go

```diff
@@ -212,6 +213,30 @@ func (cli *Cli) ResolveDependencies(ctx context.Context, projectPath string) err
 	return nil
 }
 
+var ErrPropertyNotFound = errors.New("property not found")
+
+func (cli *Cli) GetProperty(ctx context.Context, propertyPath string, projectPath string) (string, error) {
```

Comment:
Very slick!  I had to do something like this in a prototype but did not know about `help:evaluate" and ended up cracking the pom file in a much more brittle way.

Reply:
Thanks! The other nice thing about this is that it will run any pre-processor evaluation on the property as well -- not very common since these properties are generally static, but could be used for properties referencing other properties.